cesql/spec.md

```diff
@@ -402,6 +409,14 @@ sequence = 10
 `Integer x Integer`. Because the right operand of the operator is an _Integer_ and there is only one `=` definition
 which uses the type _Integer_ as the right parameter, `sequence` is cast to _Integer_.
 
+A CESQL engine MUST support the following type casts:
+
+1. `Integer -> String`
+1. `String -> Integer`
+1. `String -> Boolean`
```

Comment:
where are ones like this defined?

Reply:
Ah I had assumed that they were defined elsewhere and I was just seeking to clarify which ones had to be supported. But, looking more closely now I don't see those defined anywhere...

Let me add those